cmd/juju/secrets/grantrevoke.go

```diff
@@ -46,19 +46,24 @@ func (c *grantSecretCommand) secretsAPI() (GrantRevokeSecretsAPI, error) {
 	return apisecrets.NewClient(root), nil
 }
 
-// Info implements cmd.Command.
-func (c *grantSecretCommand) Info() *cmd.Info {
-	doc := `
+const (
+	grantSecretDoc = `
 Grant applications access to view the value of a specified secret.
-
-Examples:
-    grant-secret <secret-uri> <application>[,<application>...]
 `
+	grantSecretExamples = `
+    juju grant-secret 9m4e2mr0ui3e8a215n4g ubuntu-k8s
```

Comment:
Same here. Seems formatting is off?

Reply:
Should we also keep this in the doc `grant-secret <secret-uri> <application>[,<application>...]` so the user can understand the structure as well and what the values correspond to?

Reply:
This example is already shown in the `Usage` section. So I removed it from `Examples`.